crash-handler-process/platforms/process-win.cpp

```diff
@@ -64,14 +83,83 @@ bool Process_WIN::isCritical(void) {
 	return critical;
 }
 
+void Process_WIN::startMemoryDumpMonitoring(void) {
+	if (memorydump && memorydump->joinable()) {
+		return;
+	}
+
+	mds = OpenEvent(EVENT_ALL_ACCESS, FALSE, L"Global\\OBSMEMORYDUMPEVENT");
```

Comment:
Should the path be a value sent from osn through IPC? I'm thinking that we should try to avoid hardcoding variables as much as we can.

Reply:
changed code. 
now it will get events names and dump path in a request.